cmd2/parsing.py

```diff
@@ -236,33 +236,6 @@ def __init__(
         else:
             self.shortcuts = shortcuts
 
-        # this regular expression matches C-style comments and quoted
```

Comment:
Wow, that was a really nicely documented regex!

Reply:
Yes, it did work well.